app/views/ProfileView/index.js

```diff
@@ -81,7 +82,7 @@ class ProfileView extends React.Component {
 
 	componentWillReceiveProps(nextProps) {
 		const { user } = this.props;
-		if (user !== nextProps.user) {
+		if (isEqual(user, nextProps.user)) {
```

Comment:
You can use `equal` from `deep-equal` and not need to import `lodash/isEqual` :)

Reply:
Oops , sorry my bad, will resubmit again

Reply:
I have pushed another commit